pkgs/development/libraries/qtspell/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, enchant
+, qttools
+, pkg-config
+, withDocs ? false
+, doxygen
+, glib
+, llvmPackages_13
```

Comment:
Can we use `llvmPackages` directly? （Currently 11 and planning to upgrade to 16 https://github.com/NixOS/nixpkgs/pull/241692）